x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -106,25 +139,82 @@ export const isolationRequestHandler = function (
     caseIDs = [...new Set(caseIDs)];
 
     // create an Action ID and dispatch it to ES & Fleet Server
-    const esClient = context.core.elasticsearch.client.asCurrentUser;
+    let esClient = context.core.elasticsearch.client.asInternalUser;
```

Comment:
you can simplify the logic around this variable by defining it inside the block that is used.

Reply:
e1e3d4e2fc123b1a2dcba7f84dbe7ad23c4bdbff